inductiva/molecules/scenarios/protein_solvation/post_processing.py

```diff
@@ -112,15 +118,22 @@ def calculate_rmsf_trajectory(self,
         plt.title("RMSF per residue")
         plt.grid(True)
         plt.show()
+        duration = time.time() - start_time
+        print(f"RMSF calculation took {duration:.2f} seconds.")
```

Comment:
Why are you computing the time? Does it take that much to compute RMSF?

Reply:
For the big protein it was taking like 3 min!